jobs/clear-containers-agent-azure-ubuntu-16-04-PR/config.xml

```diff
@@ -119,15 +119,7 @@ cd tests
           <RunIfJobSuccessful>false</RunIfJobSuccessful>
           <script>#!/bin/bash&#xd;
 &#xd;
-export GOROOT=/usr/local/go&#xd;
-export GOPATH=$HOME/go&#xd;
-export PATH=$GOPATH/bin:/usr/local/go/bin:$PATH&#xd;
-&#xd;
-echo &quot;GOROOT=$GOROOT&quot;&#xd;
-echo &quot;GOPATH=$GOPATH&quot;&#xd;
-echo &quot;PATH=$PATH&quot;&#xd;
-echo &quot;USER=$USER&quot;&#xd;
-echo &quot;WORKSPACE=$WORKSPACE&quot;&#xd;
+export GOPATH=$WORKSPACE/go&#xd;
```

Comment:
Along with the change to `GOPATH` - presumably all the other changes (`GOROOT`, `PATH` etc.) were also intentional?
Can you ack that was deliberate @chavafg ?

Reply:
Hi @grahamwhaley, yes, they are intentional, I realized that those are not needed by the teardonw script, so I removed on all jobs.